src/utilities.h

```diff
@@ -197,6 +198,21 @@ const std::vector<std::string> supportedMathMLElements = {
     "arccot", "arcsinh", "arccosh", "arctanh", "arcsech", "arccsch", "arccoth", "pi", "exponentiale",
     "notanumber", "infinity", "true", "false"};
 
+/**
+ * @brief Map to convert an interface type into its string form.
+ *
+ * An internal map used to convert a Variable InterfaceType enum class member into its string form.
+ */
+static const std::map<Variable::InterfaceType, const std::string> interfaceTypeToString = {
+    {Variable::InterfaceType::NONE, "none"},
+    {Variable::InterfaceType::PRIVATE, "private"},
+    {Variable::InterfaceType::PUBLIC, "public"},
+    {Variable::InterfaceType::PUBLIC_AND_PRIVATE, "public_and_private"}};
```

Comment:
Is there a reason that the strings shouldn't be camel case, as elsewhere?

Reply:
Because they match what the spec. says.

Reply:
Makes sense :)